test/e2e-frontend/helpers/get.test.js

```diff
@@ -0,0 +1,290 @@
+const should = require('should');
+const sinon = require('sinon');
+const {startGhost} = require('../../utils/e2e-framework');
```

Comment:
Is there a reason `startGhost` was added to e2e-framework instead of using `testUtils.startGhost`?

Reply:
No, I've corrected it. Thanks!